classes/PodsInit.php

```diff
@@ -485,13 +494,16 @@ public function register_assets() {
 		}
 
 		wp_register_script(
-			'pods', PODS_URL . 'ui/js/jquery.pods.js', array(
+			'pods', PODS_URL . 'ui/js/jquery.pods.js',
```

Comment:
:no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: bf47de6c732ec4001373c6a3879cecff</sub>

Reply:
```suggestion
			'pods',
			PODS_URL . 'ui/js/jquery.pods.js',
```